dist/vuescroll-native.js

```diff
@@ -2320,16 +2320,20 @@ var update = {
     onMouseWheel: function onMouseWheel(event) /* istanbul ignore next */{
       var duration = this.mergedOptions.vuescroll.wheelScrollDuration;
       var isReverse = this.mergedOptions.vuescroll.wheelDirectionReverse;
+      var checkShiftKey = this.mergedOptions.vuescroll.checkShiftKey;
 
       var delta = 0;
       var dir = void 0;
       if (event.wheelDelta) {
-        if (event.deltaY) {
-          dir = 'dy';
-          delta = event.deltaY;
-        } else if (event.deltaX) {
-          delta = event.deltaX;
-          dir = 'dx';
+
+        if (event.deltaY || event.deltaX) {
+          if (Math.abs(event.deltaX) > Math.abs(event.deltaY)) {
```

Comment:
I still wonder why you want to compare the absoulu value?

Reply:
Because delta values have a sign and to compare them it is necessary to remove it. If for example we scroll with the trackpad in a direction between 90 and 180 degrees, deltax is positive and deltay is negative. To know if we are closer to 180 degrees (vertical scroll) than 90 (horizontal scroll), we need to remove the sign from the values to compare it.